polybar-scripts/player-spotify/player-spotify.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
At first please solve all shellcheck issues.

Reply:
What issues are you referring to?

Reply:
https://www.shellcheck.net/ probably